grapher/Models/Calculations/AccelCalculator.cs

```diff
@@ -99,6 +99,7 @@ public void Calculate(AccelChartData data, ManagedAccel accel, double starter, I
                 }
 
                 var ratio = outMagnitude / simulatedInputDatum.velocity;
+                ratio = Math.Min(int.MaxValue, Math.Max(int.MinValue, ratio));
```

Comment:
Can you please share with me a settings.json where the later conversion of this ratio to lower precision causes overflow?

Reply:
Wondering as well why you are only guarding ratio here and not slope.

Reply:
it crashed when use default configuration and: 
x=0, y>0 or x>0, y=0

Reply:
Actually this hotfix isn't the best, but it solves theese two extreme cases. I'm not even sure, that clamping to int isn't too much. The problem is actually in drawing data. Clamped should be everything, what goes into DataBind, DataBindX, DataBindXY, SetMinMaxX, etc... (something what could be converted to Decimal, without any exception). Interesting thing also is, that you could pass very low value, but after you pass 0, it breaks - or shows very big range on axes with this fix. Maybe there is some other part of code, which sets something to NaN, Infinity or very big or small value.

Reply:
Closing this as the issue is no longer seen.